### PR TITLE
Bug 1509311: Initial work on packet.net custom image

### PR DESCRIPTION
**Pull request by walac**

### CODE REVIEW

File: deploy/template/lib/systemd/system/docker-worker.service [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Taskcluster docker worker
+Requires=docker.service
+
+[Service]
+Type=notify
+ExecStart=/usr/local/bin/node /root/docker_worker/src/bin/worker.js --host test production
```
Comment (djmitche) [inline deploy/template/lib/systemd/system/docker-worker.service]: will this be `--host packet` later?

File: deploy/bin/build [hunk 2]
```diff
@@ -46,8 +46,16 @@ cmd_app() {
     --exclude='./worker-types-backup.json' \
     --exclude='./docker-worker-amis.json' \
     --exclude='./packer-artifacts.json' \
+    --exclude='docker-worker.tar.gz' \
+    --exclude='modules.tar.gz' \
+    --exclude='kernel.tar.gz' \
+    --exclude='initrd.tar.gz' \
```
Comment (djmitche) [inline deploy/bin/build]: Would it be better to whitelist the subdirectories required here?  I've been bitten multiple times by things that tar up a repo when I have some temporary file there containing secrets..

File: deploy/template/lib/systemd/system/docker-worker.service (hunk 1 above)
Comment (walac) [reply deploy/template/lib/systemd/system/docker-worker.service]: Yep, I need to work on the packet-net branch

File: deploy/bin/build (hunk 2 above)
Comment (walac) [reply deploy/bin/build]: Yep, I can do that in a follow up PR

**Pull request merged**